Dockerfile

```diff
@@ -78,8 +78,17 @@ COPY presto-password-authenticators /build/presto-password-authenticators
 COPY src /build/src
 COPY pom.xml /build/pom.xml
 
+# increase memory available to Maven
+ENV MAVEN_OPTS='-Xmx8192m'
+
+# build debug info
+RUN cat /proc/meminfo
+
+# load Maven cache as its own layer
+RUN echo "\$MAVEN_OPTS=$MAVEN_OPTS:" && cd /build && mvn de.qaware.maven:go-offline-maven-plugin:resolve-dependencies -DdownloadSources -DdownloadJavadoc -Dmaven.repo.local=.m2/repository
```

Comment:
```suggestion
RUN echo "\$MAVEN_OPTS=$MAVEN_OPTS:" && cd /build && mvn -B de.qaware.maven:go-offline-maven-plugin:resolve-dependencies -DdownloadSources -DdownloadJavadoc -Dmaven.repo.local=.m2/repository
```

Reply:
fixed in af9fdebe. Also pulled the whole heap memory specification entirely as fix was made in release repo